### DISCUSSION TITLE
How to remember mounted network shares?

### DISCUSSION THREAD

**Discussion by teacupteller** [elementary/files#2350] [Q&A]
You can mount a network share by opening Files and then clicking on "Connect Server..." in the left hand pane. 

![image](https://github.com/elementary/files/assets/122447362/57c56acd-9586-435a-ae31-ae3d82622b8c)

My problem is very time I restart Elementary OS, my shares are no longer mounted and I need to repeat this process. 

How can I keep these shares remembered permanently?

**Comment by jeremypw** [comment]
Similar to #1795.  You can create bookmarks to networked locations but they may not always be restored automatically on restarting (depends on a number of factors). You may need to re-enter the password.

**Reply by teacupteller** [reply]
Thank you. 

I’ll do better to search related threads in future.
